spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`.
```

Comment:
It'd be good to include a justification here to make it obvious why this is here. Maybe something like "instead of `protected` as Enumeration inheritance is currently not supported."